pkgs/development/ocaml-modules/uring/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, buildDunePackage
+, ocaml
+, cstruct
+, dune-configurator
+, fmt
+, optint
+, cmdliner
+, logs
+, lwt
+, mdx
+}:
+
+let
+  pname = "uring";
+
+  param = {
+    version = "0.5";
+    sha256 = "sha256-Jc17Myc/oUEYpwUVKlxYSNsAAXn3WuM5AqisFcBG/Xk=";
+  };
+in
+buildDunePackage {
+  inherit pname;
+  inherit (param) version;
+
+  minimalOCamlVersion = "4.12";
+  duneVersion = "3";
+
+  src = fetchFromGitHub {
```

Comment:
Please use `fetchurl` with the following URL: https://github.com/ocaml-multicore/ocaml-uring/releases/download/v0.5/uring-0.5.tbz
This tarball in particular includes correct version information.

Reply:
Why is fetchurl preferred over fetchFromGitHub?

Reply:
The change that I suggest is to download the released tarball instead of a snapshot of the git repository.